rdt-app/src/main/java/io/ona/rdt_app/application/RDTApplication.java

```diff
@@ -115,4 +133,33 @@ public static CommonFtsObject createCommonFtsObject() {
     private static String[] getFtsSortFields() {
        return new String[]{Constants.DBConstants.BASE_ENTITY_ID, Constants.DBConstants.ID, Constants.DBConstants.NAME};
     }
+
+
+    protected void scheduleJobsPeriodically() {
```

Comment:
After app starts then schedule the jobs immediately, then schedule them periodically

Reply:
Done.